go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/qor/qor
 
-go 1.21
+go 1.23.0
 
-toolchain go1.21.0
+toolchain go1.24.1
```

Comment:
For open-source Go packages, it is recommended to include only the go version in go.mod and avoid adding toolchain. This prevents affecting users’ Go version choices while maintaining good compatibility.